cli/cmd/install.go

```diff
@@ -241,7 +371,7 @@ func render(config installConfig, w io.Writer, options *installOptions) error {
 
 	injectOptions := newInjectOptions()
 
-	*injectOptions.proxyConfigOptions = *options.proxyConfigOptions
+	injectOptions.proxyConfigOptions = options.proxyConfigOptions
```

Comment:
This func modifies `injectOptions.proxyConfigOptions`, which will now also modify `options.proxyConfigOptions` if you make this change. Was it intended? The previous setup ensured that this func did not modify the `options` param that's passed to it. We could add a comment to that effect to make this clearer.